noxfile.py

```diff
@@ -24,8 +24,12 @@
 )
 GOOGLE_AUTH = 'google-auth >= 0.10.0'
 
+DEFAULT_PYTHON_VERSION = "3.8"
+SYSTEM_TEST_PYTHON_VERSIONS = ["2.7", "3.8"]
+UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.5", "3.6", "3.7", "3.8"]
```

Comment:
Is this library's noxfile hand-written or does synth modify it in any way? I presume the former?

Reply:
It is handwritten: `synth.py` explicitly excludes copying it.